pegasus_wrapper/artifact.py

```diff
@@ -40,7 +40,7 @@ def from_job(job: Job, output_files: Optional[Iterable[File]]) -> "DependencyNod
 
     @staticmethod
     def already_done() -> "DependencyNode":
-        return DependencyNode(job=None)
+        return DependencyNode(job=None, output_files=immutableset())
```

Comment:
What's the reasoning behind this change?

Reply:
I believe I was getting a warning by a precommit tool. There's not a default specified in the `attrib` for output_files so it needs to just be set to an empty set as it's not optional.